boards/common/blxxxpill/Makefile.features

```diff
@@ -11,3 +11,4 @@ FEATURES_PROVIDED += periph_spi
 FEATURES_PROVIDED += periph_timer
 FEATURES_PROVIDED += periph_uart
 FEATURES_PROVIDED += periph_qdec
+FEATURES_PROVIDED += periph_usbdev
```

Comment:
If you don't have time to look into, I would be fine to merge this as is with just the feature disabled and a comment. Maybe someone else would pick this up from there.

```suggestion
# TODO: Figure out why the USB driver doesn't work
# FEATURES_PROVIDED += periph_usbdev
```